packages/back-end/src/models/FeatureRevisionModel.ts

```diff
@@ -265,11 +284,14 @@ export async function updateRevision(
   changes: Partial<
     Pick<
       FeatureRevisionInterface,
-      "comment" | "defaultValue" | "rules" | "baseVersion"
+      "comment" | "defaultValue" | "rules" | "baseVersion" | "status"
```

Comment:
Don't add `status` to changes.  Instead, this function should use the `resetReview` argument that's passed in to determine what status needs to change to (if any).  This will reduce the amount of copy/pasted code and reduce the chance of errors in the future.